src/utils/yaml-utils.ts

```diff
@@ -0,0 +1,8 @@
+import DOMPurify from "isomorphic-dompurify"
+import yaml from "yaml"
+
+export const sanitizedYamlParse = (unparsedContent: string) =>
+  yaml.parse(DOMPurify.sanitize(unparsedContent))
+
+export const sanitizedYamlStringify = (prestringifiedContent: object) =>
+  DOMPurify.sanitize(yaml.stringify(prestringifiedContent))
```

Comment:
makes sense - resolved in [6962058](https://github.com/isomerpages/isomercms-backend/pull/591/commits/6962058620f19bb4e0c46c5acfb3a1ed6201fbaf)

Reply:
sorry for being a pedant but could we change the return type of `sanitizedYamlParse` to be `Record<string, unknown>`? returning `object` is dangerous because this means that callers don't have to validate any values (and the inferred type is also `any`) 

eg: 
```typescript
const x = sanitizedYamlParse("asdfsadf")
// y is inferred as `any`
const y = x.a // note that calling `x.a` is also invalid here
```

to be even more exact, we can type this as `Result<Record<string, unknown>, UnprocessableError> with a `try/catch` around the `yaml.parse` so that a parse failure is handled properly

Reply:
Added the return type `Record<string, unknown>` in [50d2198](https://github.com/isomerpages/isomercms-backend/pull/591/commits/50d21982cffa0db19e2fc4ec9f30a71f1f690135).

I initially made a commit to use neverthrow but decided that we should tackle it separately when we modify more parts of the codebase to use it.